site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1636,12 +1636,13 @@ public function getAllGradeablesIds() {
      *
      * @return array
      */
-    public function getAllElectronicGradeablesIds() {
+    public function getAllElectronicGradeables() {
         $this->course_db->query("
-          SELECT g_id, g_title
-          FROM gradeable INNER JOIN electronic_gradeable USING (g_id)
-          WHERE eg_scanned_exam=FALSE and eg_has_due_date=TRUE
-          ORDER BY eg_submission_due_date ASC
+          late_days_for_excused
+            SELECT gradeable.g_id, g_title, eg_submission_due_date
+            FROM gradeable INNER JOIN electronic_gradeable 
+                ON gradeable.g_id = electronic_gradeable.g_id
+            WHERE g_gradeable_type=0 ORDER BY g_grade_released_date DESC
```

Comment:
The first line of the query probably shouldn't be there? (`late_days_for_excused`)

It also looks like this will change the result set here to now also include rows where `eg_scanned_exam=TRUE` and `eg_has_due_date=FALSE`?

Reply:
I have fixed them now. I think it is good to go now.